shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -15,9 +15,6 @@
 #import "LogSend.h"
 
 @interface AppDelegate ()
-
-@property (nonatomic, strong) LogSend * logSender;
```

Comment:
turns out this doesn't work. When JS calls a native method it'll instantiate the object itself with [obj new] so it won't use our pre-existing instance. Therefore no reason to store this in app delegate